cime_config/buildnml

```diff
@@ -476,11 +476,11 @@ def buildnml(case, caseroot, component):
     for filename in glob.glob(os.path.join(confdir, "*modelio*")):
         shutil.copy(filename, rundir)
 
-    # copy fd.yaml to rundir
+    # copy fd_cesm.yaml to rundir
     cimeroot = case.get_value("CIMEROOT")
     fd_dir = os.path.join(cimeroot, "src","drivers","nuopc","mediator")
-    filename = os.path.join(fd_dir,"fd.yaml")
-    shutil.copy(filename, rundir)
+    filename = os.path.join(fd_dir,"fd_cesm.yaml")
```

Comment:
I think it is better to make this part as generic as possible and it helps to adapt s2s application easily in the near future.

coupling_mode = case.get_value('coupling_mode')
if coupling_mode == 'cesm':
     filename = os.path.join(fd_dir,"fd_cesm.yaml")
else:
    filename = os.path.join(fd_dir,"fd_nems.yaml")